File: Java OO/Empresa.java
public class Empresa {
    private String nome;
    private String localizacao;

    public Empresa(String nome, String localizacao) {
        this.nome = nome;
        this.localizacao = localizacao;
    }

    public void exibir() {
        System.out.println("Empresa: " + nome + ", Localização: " + localizacao);
    }
}

File: JavaScript OO/empresa.js
class Empresa {
    constructor(nome, localizacao) {
        this.nome = nome;
        this.localizacao = localizacao;
    }

    exibir() {
        console.log(`Empresa: ${this.nome}, Localização: ${this.localizacao}`);
    }
}

module.exports = Empresa;
